src/components/Footer.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import { IconButton } from '@frequency-chain/style-guide';
+  import XLogo from './icons/XLogo.svelte';
+  import DiscordLogo from './icons/DiscordLogo.svelte';
+  import FrequencyLogo from './icons/FrequencyLogo.svelte';
+
+  export let privacyHref: string = 'https://www.frequency.xyz/privacy';
+  export let type: 'light' | 'dark' = 'light';
+
+  let fgColorText = {
+    dark: 'text-white',
+    light: 'text-black',
+  }[type];
+  let fgColorBg = {
+    dark: 'bg-white',
+    light: 'bg-black',
+  }[type];
```

Comment:
The reason is the same as for Faucet:  the link to the Privacy Policy is '/privacy', so that meant a choice between importing a Privacy Policy page that might go out of date, or importing the Footer and pointing to `https://www.frequency.xyz/privacy`, and I chose the latter.